src/macros.rs

```diff
@@ -199,7 +199,7 @@ macro_rules! e {
 // cfg completely.
 // FIXME(ctest): ctest can't handle `$(,)?` so we use `$(,)*` which isn't quite correct.
 cfg_if! {
-    if #[cfg(feature = "const-extern-fn")] {
```

Comment:
consider removing the feature from the default features (in this commit, so it is backported) and also updating the comment in `src/macros.rs` in this commit (so it no longer mentions this feature, also so it is backported).

Reply:
Reasonable 👍 changed, thanks for taking a look.